Sources/FoundationNetworking/URLCredential.swift

```diff
@@ -91,9 +114,14 @@ open class URLCredential : NSObject, NSSecureCoding, NSCopying {
         guard aCoder.allowsKeyedCoding else {
             preconditionFailure("Unkeyed coding is unsupported.")
         }
-        
-        aCoder.encode(self._user._bridgeToObjectiveC(), forKey: "NS._user")
-        aCoder.encode(self._password._bridgeToObjectiveC(), forKey: "NS._password")
+
+        if let user = self._user {
+            aCoder.encode(user._bridgeToObjectiveC(), forKey: "NS._user")
```

Comment:
Let's make sure we also change `init?(coder:)` above

```
let encodedUser = aDecoder.decodeObject(forKey: "NS._user") as! NSString
let encodedPassword = aDecoder.decodeObject(forKey: "NS._password") as! NSString
```

to no longer force-unwrap, since we may no longer be encoding `nil` user or password, e.g. we could do

```
if let encodedUser = aDecoder.decodeObject(forKey: "NS._user") as? NSString {
  self._user = bridgeString(encodedUser)!
}
...
```